ci/pipeline.yaml

```diff
@@ -10,6 +10,14 @@ resource_types:
       username: ((harbor.username))
       password: ((harbor.token))
 
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: harbor-repo.vmware.com/dockerhub-proxy-cache/teliaoss/github-pr-resource
```

Comment:
This si very interesting. So is this image watching the github repository and then interacting with the GH API?

Reply:
This (inside the `resource_types` section, is how we add new kinds of resources to concourse. Think of it like adding a plugin. Concourse has lots of built-in types (like, git, s3, etc...), but a few need to be added. In this case, we're adding the ability to monitor pull requests from github.

This definition itself doesn't pull PRs from github, but it adds the functionality. The real place that pulls PRs is in the `resources` section, where we define the `pull-requests` resource.

Reply:
Cool!

Very interesting way to not to rely on Github sending a webhook to your CI